### ISSUE TITLE
Add Outreachy application template

### ISSUE THREAD

**Issue by yochannah** [intermine/gsoc#19]
We should add an outreachy proposal template as well as a GSoC one. 

Task: 

Make a copy of this: 
https://docs.google.com/document/d/1rvZG5SDqdvSTBxLrVy0JbVbs0A42rHToJwxHd89Hftc/edit

Remove the GSoC logo and re-brand to be outreachy where appropriate. Let us know if you think anything else needs to be changed. Once it is done, share with a mentor and/or @yochannah and make a PR to update http://intermine.org/internships/guidance/students-applying-outreachy/ with a link to the new PR (code is here: https://github.com/intermine/internships/blob/master/guidance/students-applying-outreachy.md )

**Comment by PrastutiUpadhaya** [comment]
Hi @yochannah Can I work on this?
